backend/code/archipelag/message/views.py

```diff
@@ -12,6 +13,22 @@
 from archipelag.market.settings import POINTS_RULES
 from archipelag.ngo.models import NgoUser
 
+from archipelag.event_log.models import EventLog
+from archipelag.event_log.serializers import EventLogSerializer
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+
+
+class MessageList(generics.ListAPIView):
+    permission_classes = (IsAuthenticated,)
+    serializer_class = MessageSerializer
+
+    def get_queryset(self):
```

Comment:
Get one element is not a list of elements. You should use RetrieveAPIView or ModelViewSet (has a list/single/post/put and delete in one class)

Reply:
I dont get here one element. I get list of messages thats belong to one market

Reply:
>_< ok.